src/templates/typescript/registerPartials.ts

```diff
@@ -4,15 +4,21 @@ import complexMemberTemplate from "./classes/complexMemberTemplate";
 import complexParseTemplate from "./classes/complexParseTemplate";
 import primitiveParseTemplate from "./classes/primitiveParseTemplate";
 import choiceTypeTemplate from "./classes/choiceTypeTemplate";
+import interfaceChoiceTypeTemplate from "./interfaces/interfaceChoiceTypeTemplate";
 import interfaceMemberTemplate from "./interfaces/interfaceMemberTemplate";
 import primitiveMemberTemplate from "./interfaces/primitiveMemberTemplate";
+import parseSingleMemberTemplate from "./classes/parseSingleMemberTemplate";
+import parseChoiceMemberTemplate from "./classes/parseChoiceMemberTemplate";
 
 Handlebars.registerPartial("classImport", classImportTemplate);
 Handlebars.registerPartial("complexMember", complexMemberTemplate);
 Handlebars.registerPartial("interfaceMember", interfaceMemberTemplate);
 Handlebars.registerPartial("primitiveMember", primitiveMemberTemplate);
 Handlebars.registerPartial("choiceType", choiceTypeTemplate);
+Handlebars.registerPartial("interfaceChoiceType", interfaceChoiceTypeTemplate);
 Handlebars.registerPartial("primitiveParse", primitiveParseTemplate);
 Handlebars.registerPartial("complexParse", complexParseTemplate);
+Handlebars.registerPartial("parseSingleMember", parseSingleMemberTemplate);
+Handlebars.registerPartial("parseChoiceMember", parseChoiceMemberTemplate);
```

Comment:
Codacy found an issue: [Unsafe call of an any typed value.](https://app.codacy.com/gh/MeasureAuthoringTool/model-info-parser/pullRequest?prid=5923420)